static/lang/re-en.json

```diff
@@ -2320,6 +2320,7 @@
                 },
                 "TalismanEsoterica": "Talisman Esoterica"
             },
+            "ThreeActions": "Three Actions",
```

Comment:
I meant the whole thing under the same structure. So that it's `PF2E.SpecificRule.Actions.TwoActions`.

Reply:
Oops, my bad